sherpa/astro/xspec/__init__.py

```diff
@@ -874,18 +874,28 @@ class XSbmc(XSAdditiveModel):
 
     The model is described at [1]_.
 
+    .. note:: Deprecated in Sherpa 4.9.0
+              ``logA`` will be removed in the next release as it has been
+              replaced by ``log_A``, to match the XSPEC naming convention.
+
     Attributes
     ----------
     kT
         The temperature of the thermal photon source in keV.
     alpha
         The energy spectral index.
-    logA
+    log_A
         The log of the A parameter: see [1]_ for more details.
     norm
         The normalization of the model: see [1]_ for an explanation
         of the units.
 
+    Notes
+    -----
+    In Sherpa 4.9.0 the ``logA`` parameter has been renamed to ``log_A``.
+    It can still be accessed using ``log_A`` for the time being, but this
```

Comment:
I believe it should say `logA` on this line. Also, the version should be changed from Sherpa 4.9.0 to Sherpa 4.10.0 (we still have to fix the ciao/sherpa versioning scheme, I'll send a proposal for that). And a few lines above, I am relaxing the statement "`logA` will be removed in the next release" with a "might be removed in future releases".